packages/build/README.md

```diff
@@ -101,6 +100,42 @@ Now you run the scripts, such as:
 npm run build
 ```
 
+5. Run code coverage reports
+- lb-nyc
+
+  By default, `lb-nyc` is a plain wrapper for https://github.com/istanbuljs/nyc
+
+  To customize the configuration:
+
+  - Create `.nycrc` in your project's root
+    directory
+
+    ```json
+    {
+      "include": [
+        "dist"
+      ],
+      "exclude": [
+        "dist/__tests__/"
+      ],
+      "extension": [
+        ".js",
+        ".ts"
+      ],
+      "reporter": [
+        "text",
+        "html"
+      ],
+      "exclude-after-remap": false
+    }
+    ```
+
+  - To run code coverage with every test execution update your package.json:
+  ```json
+    "coverage": "lb-nyc",
+    "test": "npm run coverage lb-mocha \"dist/__tests__\"",
```

Comment:
`npm run coverage -- lb-mocha \"dist/__tests__\"`?

Reply:
In loopback-next, we define `coverage` script to open the coverage report, and run the tests from a `precoverage` script. See here:

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L43-L44

There is also a special script to run the tests & report the coverage to Coveralls:

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L42

Here is how we run `lb-nyc` (please replace `node packages/build/bin/run-nyc` with `lb-nyc`):

https://github.com/strongloop/loopback-next/blob/c8fb8058c375fe22b0c64988c7ff91c1024084ed/package.json#L57-L58

I would like us to stay consistent and use the same approach in all places (`loopback-next` scripts in `package.json`, documentation for `@loopback/build`).